superset-frontend/src/explore/components/ExploreChartHeader/index.jsx

```diff
@@ -169,53 +159,42 @@ export class ExploreChartHeader extends React.PureComponent {
         }
       })
       .catch(() => {});
-  }
+  };
 
-  getSliceName() {
-    const { sliceName, table_name: tableName } = this.props;
+  const getSliceName = () => {
     const title = sliceName || t('%s - untitled', tableName);
 
     return title;
-  }
+  };
```

Comment:
This function can be cleaned to one line like this:
`const getSliceName = () => sliceName || t('%s - untitled', tableName);`